mitodl/hugo-course-publisher
# Update sample markdown with UID only menu identifiers

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes mitodl/hugo-course-publisher#104

#### What's this PR do?
Removes the `short_url` property from menu identifiers.  The unpredictable nature of these strings means that they sometimes break the nav due to unsupported characters for use in HTML ID properties.

#### How should this be manually tested?
Run the site or visit the deploy preview.  Visit "Single Variable Calculus."  The "differentiation" menu item should now expand & collapse.

#### Screenshots (if appropriate)
![Screen Shot 2020-04-02 at 5 44 25 PM](https://user-images.githubusercontent.com/12089658/78302938-c8aa9780-7509-11ea-8812-c0ea0425a10f.png)
